### PR TITLE
Adds tweenBuffer

### PR DESCRIPTION
**Pull request by NicholasLancey**
## Description

QA encountered an issue where if you clicked on the clickable card directly after it closed, it would reopen in a broken state. 

## Solution

@liamegan and I added a 1 second buffer to prevent the card from being reopened straight away. 

Will discuss this one in the dev meeting on Friday.

### CODE REVIEW

File: src/wtc-perspective-card.js [hunk 1]
```diff
@@ -816,7 +818,10 @@ class ClickablePerspectiveCard extends PerspectiveCard {
 
   // Toggle the enlarged flag on click
   onClick(e) {
-    if (window.clickablePerspectiveCard_initialtouch === e.pointerId) {
+    if (
+      window.clickablePerspectiveCard_initialtouch === e.pointerId &&
+      this._tweenBuffer == false
```
Comment (liamegan) [inline src/wtc-perspective-card.js]: Please change this to `=== false`

**Review by liamegan — CHANGES_REQUESTED** [review]

File: src/wtc-perspective-card.js [hunk 2]
```diff
@@ -981,6 +986,12 @@ class ClickablePerspectiveCard extends PerspectiveCard {
    * @default false
    */
   set tweening(value) {
+    if (value !== this._tweening) {
+      this._tweenBuffer = true;
```
Comment (marlonmarcello) [inline src/wtc-perspective-card.js:990]: If I read the problem correctly, the issue is clicking on the card as soon as it closes right?
Shouldn't this buffer be turned ON and then after a second be turned OFF when setting `_tweening` to `false` then instead?

**Comment by liamegan** [comment]
Depending on the implementation - ie if you were to extend it to allow
clicking of the card when enlarged - then this problem would still present
itself. This fix just protects against both potentials.


---------------------------------------------------------------
Liam Egan  |  We The Collective Design Inc.

37 Dunlevy Ave.
Vancouver BC V6A 3A3

Phone: +1 (604) 363-6466

www.wethecollective.com

**Comment by marlonmarcello** [comment]
Got it!

File: src/wtc-perspective-card.js (hunk 1 above)
Comment (marlonmarcello) [reply src/wtc-perspective-card.js]: Will merge after this one.

**Comment by NicholasLancey** [comment]
All good to go @marlonmarcello

**Comment by marlonmarcello** [comment]
Thank you guys!

**Review by marlonmarcello — APPROVED** [review]

**Pull request merged**